src/UNL/Peoplefinder/Driver/OracleDB.php

```diff
@@ -260,14 +259,12 @@ public function fixLDAPEntries(array $entries)
                 $entries[$key]['mail'] = $value;
             }
 
-            // Use the affiliations from Oracle. There is additional processing being done on the UNL_AFFILIATIONS_00
-            // view before it reaches us to remove affiliations associated with "Directory Order=NL" appointments.
-            if (!empty($row['AFFILIATION'])) {
-                $affiliations = explode(';', $row['AFFILIATION']);
-                $affiliations = array_unique(array_map('strtolower', $affiliations));
-                $value = new UNL_Peoplefinder_Driver_LDAP_Multivalue($affiliations);
-
-                $entries[$key]['edupersonaffiliation'] = $value;
+            //prepares the BIODEMO_IDs to be used in the affiation query
+            if(!empty($row['BIODEMO_ID'])){
```

Comment:
Coding standards: Let's do a space before and after the parenthesis in if and foreach clauses

Reply:
I will add those